### PR TITLE
[Release] Jumplinks, Resource Collections, Menu, Updated Documentation

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé

Included in this release:
- Jumplinks Feature (F3)
- Resource Collections Feature (F2)

## ⚠️ This PR can be tested through the following URL(s):
Main - https://staging.learning-resources.cdssandbox.xyz/
EN - https://en.staging.learning-resources.cdssandbox.xyz/
FR - https://fr.staging.learning-resources.cdssandbox.xyz/fr/

# 🚀  Release Checklist

Item | Checked / Reviewed by | Status | Notes
-- | -- | -- | --
Metrics - Google Analytics working and collecting data |  @daine @hillaryl   | Not started | 
Content - Contentful |   | Not started |
Content - English | @anikbrazeau  | Not started |
Content - French | @anikbrazeau  | Not started |
Infra - Domains OK | @daine   | ☀️ Ready |
Infra - Amplify OK | @daine | ☀️ Ready  |
Monitoring (Uptime Robot)| @daine | ☀️ Ready  |  [Status page dashboard](https://stats.uptimerobot.com/gP5gohqEVQ)
Accessibility - aXe DevTools | @daine @dylanzheng94   | Not started |
Accessibility - Lighthouse checks | @daine @dylanzheng94 | Not started |
Accessibility - WCAG 2.0 AA | @emanelfy | Not started | The intention is to achieve AAA for full launch
Infra - Content Security Policy | @daine  | ☀️ Ready |

# After merge, do this:
From your terminal, make sure you are in the `main` branch, then type
```bash
git tag -a v0.2.0 -m <release notes>
git push --tags
```
This bumps up the release version

### CODE REVIEW

File: app/pages/resource/_resource.vue [hunk 1]
```diff
@@ -137,6 +165,10 @@ export default {
       fr: { resource: frRouteParam },
     })
 
+    // const name = "Dylan"
+    // console.log(name);
```
Comment (daine) [inline app/pages/resource/_resource.vue:169]:
```suggestion
```

File: app/locales/en.json [hunk 2]
```diff
@@ -42,5 +42,6 @@
     "resource": "resource/",
     "legal": "legal/"
   },
-  "canada_wordmark_alt": "Symbol of the Government of Canada"
+  "canada_wordmark_alt": "Symbol of the Government of Canada",
+  "jump_to": "Jump to"
```
Comment (anikbrazeau) [inline app/locales/en.json:46]:
```suggestion
  "jump_to": "On this page"
```

File: .github/PULL_REQUEST_TEMPLATE/release_template.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ FR - https://fr.staging.learning-resources.cdssandbox.xyz/fr/
 
 Item | Checked / Reviewed by | Status | Notes
 -- | -- | -- | --
-- [] Metrics - Google Analytics working and collecting data |  @daine @hillaryl   | Not started | [Prioritized for next release](https://github.com/cds-snc/resources-ressources/issues/154)
+Metrics - Google Analytics working and collecting data |  @daine @hillaryl   | Not started | [Prioritized for next release](https://github.com/cds-snc/resources-ressources/issues/154)
 Content - Contentful |   | ☀️ Ready |
 Content - English | @anikbrazeau  | ☀️ Ready |
 Content - French | @anikbrazeau  | ☀️ Ready |
```
Comment (anikbrazeau) [inline .github/PULL_REQUEST_TEMPLATE/release_template.md:23]:
```suggestion
Content - Contentful |   | Not started |
Content - English | @anikbrazeau  | Not started |
Content - French | @anikbrazeau  | Not started |
```

**Comment by anikbrazeau** [comment]
@dylanzheng94 @daine  Wondering about ordering when it comes to jump to links. Should we consider putting them below H1 since they highlight all the H2s? What do you think?

Screen Shot 2022-09-13 at 5 25 16 PM

**Comment by anikbrazeau** [comment]
Curious what the behavior might be if there are no H2s? Does the anchor link component, disappear in that use case?

File: app/pages/resource/_resource.vue [hunk 4]
```diff
@@ -9,29 +9,58 @@
...
-          <ul class="mt-5 grid grid-cols-1 gap-2">
-            <!-- Resource card --------------------------------------------------------------------------------------------->
-
-            <li v-for="resource in relatedResources" :key="resource.title">
-              <ResourceListItem :resource="resource"> </ResourceListItem>
-            </li>
-          </ul>
+      <div class="max-w-full">
+        <div class="flex flex-col lg:flex-row items-start gap-8">
+          <!-- MVP Feature 3: Content jump links --------------------------------------------------------------------->
```
Comment (daine) [inline app/pages/resource/_resource.vue:14]: Let's try to fix this soon as we can --issue #233

**Comment by anikbrazeau** [comment]
Some issues we might want to tackle before releasing to production: 
- https://github.com/cds-snc/resources-ressources/issues/251
- https://github.com/cds-snc/resources-ressources/issues/248

**Comment by daine** [comment]
Found a suggestion for improving on accessibility:
Screen Shot 2022-09-21 at 10 25 08 AM
Screen Shot 2022-09-21 at 10 31 44 AM

**Comment by anikbrazeau** [comment]
I wonder if our pages that are longest, and might benefit most from the anchor "jump to" links, might actually be the [Privacy](https://en.staging.learning-resources.cdssandbox.xyz/legal/privacy) and [Terms](https://en.staging.learning-resources.cdssandbox.xyz/legal/terms) pages? Food for thought.

Screen Shot 2022-09-21 at 4 37 33 PM

Screen Shot 2022-09-21 at 4 37 40 PM

**Comment by anikbrazeau** [comment]
I wonder if it'd be possible to add some space/padding to the text when jumping to a particular section? For ease of readability, maybe it could be aligned with the start of "On this page" rather than the very top of the browser window? 

Screen Shot 2022-09-21 at 9 59 16 PM

**Comment by anikbrazeau** [comment]
Fixes #321

**Review by anikbrazeau — APPROVED** [review]
From what I can tell it looks good in staging. 👍 🚀

**Pull request merged**